fedora/cinc/install_pkg.sh

```diff
@@ -114,6 +114,8 @@ yum -y --skip-broken install\
 	which\
 	fping\
 	perf\
+	git\
+    lksctp-tools-devel\
```

Comment:
NIT: space. Sorry if I missed this on my previous review. :(

Reply:
Done